src/pages/NewMemberConsole.js

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect, useCallback } from 'react';
+import { div, h, img } from 'react-hyperscript-helpers';
+import { DAR } from '../libs/ajax';
+import SearchBar from '../components/SearchBar';
+import { Notifications, getElectionDate, processElectionStatus, darSearchHandler } from '../libs/utils';
+import { Styles} from '../libs/theme';
```

Comment:
minor nit: spacing alignment
```suggestion
import { Styles } from '../libs/theme';
```